variables.tf

```
@@ -0,0 +1,23 @@
+variable "aws_account_alternate_contacts" {
```

Following along with the update to the variable name:

```suggestion
variable "account_alternate_contacts" {
```

---

updated